src/fiatExchanges/UniquePaymentSection.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+
+import { useTranslation } from 'react-i18next'
+import { Image, StyleSheet, Text, View } from 'react-native'
+import Touchable from 'src/components/Touchable'
+import { CICOFlow, FetchProvidersOutput, PaymentMethod } from 'src/fiatExchanges/utils'
+import colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+
+export interface UniquePaymentSectionProps {
+  paymentMethod: PaymentMethod
+  uniqueProvider: FetchProvidersOutput | null
+  setNoPaymentMethods: React.Dispatch<React.SetStateAction<boolean>>
+  flow: CICOFlow
+}
+
+export function UniquePaymentSection({
+  paymentMethod,
+  uniqueProvider,
+  setNoPaymentMethods,
+  flow,
+}: UniquePaymentSectionProps) {
+  const { t } = useTranslation()
+
+  if (!uniqueProvider || uniqueProvider.restricted) {
+    setNoPaymentMethods(true)
+    return null
+  }
+
+  return (
+    <View style={styles.container}>
+      <Touchable>
+        <View style={{ ...styles.innerContainer, paddingVertical: 27 }}>
+          <View style={styles.left}>
+            <Text style={styles.category}>
+              {paymentMethod === PaymentMethod.Coinbase
+                ? t('selectProviderScreen.coinbase')
```

Comment:
`Coinbase Pay` will get translated into different languages when you use a string through this translator function. Do we actually want that to get translated since its a brand name?

Reply:
Probably not. I'll take it out of translation.